### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by RainerBielefeld**
This is the German translation for the Help topic "'Web Space"

### CODE REVIEW

File: pages/lists/list-user/web-space/de.text [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@title = 'Web Space'
+(Gemeinsame Dokumente)
```
Comment (kradan) [inline pages/lists/list-user/web-space/de.text]: this line won't appear in the title and should be removed or added to the headline below

File: pages/lists/list-user/web-space/de.text [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+h2. Wie kann ich eine Datei hochladen oder ansehen?
+
+Vollständige Informationen dazu auf [[shared-web-space]].
+
+h2.  Warum gelingt es mir nicht, eine Datei hochzuladen?
```
Comment (kradan) [inline pages/lists/list-user/web-space/de.text]: the excess space should be removed (just as in the original)

File: pages/lists/list-user/web-space/de.text [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+Es gibt verschiedene mögliche Gründe, warum das Hochladen scheitert:
+
+* Der Dateiname darf keine Leerzeichenenthalten. Bitte vor dem Hochladen Datei umbenennen, Leerzeichen durch ein zulässiges Zeichen wie wie Bindestrich '-' oder Unterstrich '_' ersetzen!
```
Comment (kradan) [inline pages/lists/list-user/web-space/de.text]: missing space after 'Leerzeichen"

File: pages/lists/list-user/web-space/de.text [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+* Eventuell fehlen Dir die erforderlichen Rechte zum Hochladen von Dateien? In nicht-öffentlichen Arbeitsgruppen-Listen ist standardmäßig das Hochladen erlaubt.In allen anderen Listen ist das Hochladen nur Nutzern mit der Nutzerrolle „Editor“ erlaubt. Diese Voreinstellungen können vom Listen-Administrator durch Anpassung der Listen-Konfiguration geändert werden.
```
Comment (kradan) [inline pages/lists/list-user/web-space/de.text]: missing space after sentence: ".In"

so far roles like "Nutzer" have been gendered to the the gender-neutral form if possible ("Nutzer*innen")

File: pages/lists/list-user/web-space/de.text (hunk 2 above)
Comment (RainerBielefeld) [reply pages/lists/list-user/web-space/de.text]: Done

File: pages/lists/list-user/web-space/de.text (hunk 1 above)
Comment (RainerBielefeld) [reply pages/lists/list-user/web-space/de.text]: "this line" means line 2? It's intended not to show that in the title line, but in normal text size below the title. It's simply a more applicable wording than "web space". In the final view it should look like

### Web Space
(Gemeinsame Dokumente)

File: pages/lists/list-user/web-space/de.text (hunk 3 above)
Comment (RainerBielefeld) [reply pages/lists/list-user/web-space/de.text]: Done

File: pages/lists/list-user/web-space/de.text (hunk 4 above)
Comment (RainerBielefeld) [reply pages/lists/list-user/web-space/de.text]: Done!

File: pages/lists/list-user/web-space/de.text (hunk 1 above)
Comment (kradan) [reply pages/lists/list-user/web-space/de.text]: it would look like this: https://share.riseup.net/#xCQWJBd0q7r5iOGci3qkXQ

Comment (RainerBielefeld) [reply pages/lists/list-user/web-space/de.text]: Hm, that is not intended, "(Gemeinsame Dokumente)" should appear below title and above list. Not below list.
My amber knowledge is too poor, can you shift "(Gemeinsame Dokumente)"  to the intended place directlybelow the title?

Comment (kradan) [reply pages/lists/list-user/web-space/de.text]: Try: ```@summary = 'Gemeinsame Dokumente'```

**Comment by kradan** [comment]
It may seem minor but to condense the git log a bit, [commits can be squashed](https://davidwalsh.name/squash-commits-git) into one commit after fixing all the issues. It also enables to change the commit message to something like `de: lists/list-user/web-space` etc.

File: pages/lists/list-user/web-space/de.text [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+h2. Wie kann ich eine Datei hochladen oder ansehen?
+
+Vollständige Informationen dazu auf [[shared-web-space]].
+
+h2. Warum gelingt es mir nicht, eine Datei hochzuladen?
+
+Es gibt verschiedene mögliche Gründe, warum das Hochladen scheitert:
+
+* Der Dateiname darf keine Leerzeichen enthalten. Bitte vor dem Hochladen Datei umbenennen, Leerzeichen durch ein zulässiges Zeichen wie wie Bindestrich '-' oder Unterstrich '_' ersetzen!
```
Comment (kradan) [inline pages/lists/list-user/web-space/de.text]: > wie wie

**Comment by RainerBielefeld** [comment]
I wonder when the translation will find it's way into the website?

**Comment by disturbio** [comment]
On 03/03/2020, RainerBielefeld wrote:
> I wonder when the translation will find it's way into the website?

Hi,

The main website is deployed by hand, and we don't have a fixed 
day/time to do it. I'm going to try to ensure it happens at least 
twice a week.

Thanks for all your help and I'm very sorry how slow we are 
sometimes. We are trying to change this, since waiting sucks and 
forever waiting is extremely disappointing.

Again, thank you for your contributions.

In solidarity,
g.

**Pull request merged**